spec/src/main/asciidoc/rest-endpoints.adoc

```diff
@@ -168,6 +168,207 @@ myClass_myMethod_seconds_max{mp_scope="vendor"} 0.05507899 <3>
+
+[[percentile-configuration]]
+==== Percentile customization for histogram and timers
+
+The percentiles that are tracked and are output can be overridden and customized by using the MicroProfile Config property `mp.metrics.distribution.percentiles`. The property follows these rules:
+
+* The property accepts a semi-colon separated set of values that consists of a metric name followed by an equals sign (`=`) and comma separated percentile values _(See <<percentiles-examples,examples>> below)_.
+* The asterisk (`*`) can be used as a wildcard at the end of the metric name.
+* The percentile values are decimal values between 0.0 and 1.0 inclusively. 
+* Setting the property with a metric name but no percentile values disables percentile output for that metric.
+* Setting the property without any value disables percentiles for all histogram and timer metrics.
+* Values defined later take precedence over values before (i.e., right to left precedence).
+
+[[percentiles-examples]]
+.MicroProfile Config `mp.metrics.distribution.percentiles` property examples
+----
+//alpha.histogram will publish the 30th and 40th percentile and alpha.timer will publish the 50th and 80th percentile
+mp.metrics.distribution.percentiles=alpha.histogram=0.3,0.4;alpha.timer=0.5,0.8
+
+//any timer or histogram that match alpha.* will publish the 60th percentile
+mp.metrics.distribution.percentiles=alpha.*=0.6
+
+//any timer or histogram that match alpha.* will publish the 60th percentile and alpha.test.histogram which will only publish the 40th percentile due to precedence
+mp.metrics.distribution.percentiles=alpha.*=0.6;alpha.test.histogram=0.4
+
+//disables percentiles for all histogram and timer metrics
+mp.metrics.distribution.percentiles=*=
+
+//disables percentiles for all histogram and timer metrics
+mp.metrics.distribution.percentiles=
+----
+
+NOTE: Histograms and Timers that do not match the metric name definitions of the `mp.metrics.distribution.percentiles` property will output the default 50th, 75th, 95th, 99th and 99.9th percentiles.
+
+==== Defining histogram-buckets for histograms and timers
+
+The MicroProfile Metrics runtime offers MicroProfile Config properties to enable and explicitly set (cumulative) histogram-buckets for histograms and timers.
+The property `mp.metrics.distribution.timer.buckets` applies to timer metrics and the property `mp.metrics.distribution.histogram.buckets` applies to histograms. These two properties follow these rules unless explicitly describing a specific property:
+
+* The property accepts a semi-colon separated set of values that consists of a metric name followed by an equals sign (`=`) and comma separated bucket values:
+** `mp.metrics.distribution.histogram.buckets` accepts a comma separated list of decimal and integer values greater than 0 _(See <<histogram-bucket-sample,examples>> below)_.
```

Comment:
should we say float here, rather than decimal?

Reply:
fixed.